charts/tempo-distributed/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 0.20.0
```

Comment:
This was just updated in another PR. Could you please rebase/merge main? Thanks!

Reply:
Thank you @annanay25, I've fixed the conflict.